pkg/engine/rule/r0002_unexpected_file_access.go

```diff
@@ -58,6 +62,25 @@ func (rule *R0002UnexpectedFileAccess) ProcessEvent(eventType tracing.EventType,
 		return nil
 	}
 
+	shouldIgnoreMounts := fmt.Sprintf("%v", rule.GetParameters()["ignoreMounts"]) == "true"
```

Comment:
1. the `Sprintf` looks weird, why do we need that?
2. Can we evaluate this once per rule creation? We should not burn any unnecessary CPU cycles in `ProcessEvent`

Reply:
1. @Bezbran Recommended that I use it to make sure the value will be string.
2. Good point.

Reply:
@Bezbran Then we need to change the design on get and set params. Or I am missing something?

Reply:
We need to store this boolean value inside the R0002UnexpectedFileAccess, when calling SetParameters implementation.
This means you should implement `func (rule *R0002UnexpectedFileAccess) SetParameters(...){...}` which will call the generic SetParameters but will implement the logic which is specific to `R0002UnexpectedFileAccess`. 
TTM if you wish.

Reply:
I am changing the code to pass params to the init of a rule